lib/sensors/MoistureSensor.cpp

```diff
@@ -0,0 +1,21 @@
+#include "mbed.h"
+#include <I2C.h>
+#include "MoistureSensor.h"
+
+MoistureSensor::MoistureSensor(PinName sda, PinName scl, char* data, int address){
+    I2C i2c(sda, scl);    
+    data_array = data;
```

Comment:
prefer to use initializer lists. https://en.cppreference.com/w/cpp/language/initializer_list

Reply:
Will be changed in later commit